web/pdf_sidebar.js

```diff
@@ -104,7 +106,7 @@ var PDFSidebar = (function PDFSidebarClosure() {
     reset: function PDFSidebar_reset() {
       this.isInitialViewSet = false;
 
-      this.close();
+      this._hideUINotification(null);
```

Comment:
I think it's there for the situation where you have opened a document with attachments/outlines and open a new document without attachments/outlines. You may be right though, because if there are attachments and `render` is called to add a new one (leading to `reset` being called), we do not want it to close (which I think will happen if `close` is put back), in which case I'm fine with leaving it out. Could you test that just to be sure?